DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -147,6 +148,7 @@ FROM Tutorials
                      AND Tutorials.TutorialID = @tutorialId
                      AND Customisations.Active = 1
                      AND CustomisationTutorials.Status = 1
+                     AND Tutorials.ArchivedDate IS NULL

```

Comment:
What about if the section is achived?

Reply:
Yep, I hadn't thought of that! Fixed in #154 🙂